client/asset/eth/eth.go

```diff
@@ -314,6 +316,8 @@ type TokenWallet struct {
 	parent   *assetWallet
 	approval atomic.Value
 	token    *dexeth.Token
+
+	asset.TxFeeEstimator
```

Comment:
What's going on with embedding an interface in the struct like this?

Reply:
`btc` implements `interfaces`  assigning them to a field in `intermediaryWallet`. I noticed `eth` && `dcr` do it differently.

Reply:
It's not analogous to this.
`ETHWallet` and `TokenWallet` implement the `asset.TxFeeEstimator` interface already.
Making an emebedded **field** does nothing.